common/browser_context.go

```diff
@@ -502,7 +502,7 @@ func (b *BrowserContext) ClearCookies() error {
 	clearCookies := storage.
 		ClearCookies().
 		WithBrowserContextID(b.id)
-	if err := clearCookies.Do(b.ctx); err != nil {
+	if err := clearCookies.Do(cdp.WithExecutor(b.ctx, b.browser.conn)); err != nil {
```

Comment:
Should we add a test for this functionality? Maybe it also makes sense to use it in the cookies example if you think it fits.

Reply:
Good points 👍 See: https://github.com/grafana/xk6-browser/pull/1040/commits/1297e3b0b9d36accb0f47971607a8835bbdd782a and https://github.com/grafana/xk6-browser/pull/1040/commits/019a02db8d24221c750610297f252179ed797bf9.